sphinx_book_theme/theme.conf

```
@@ -13,6 +13,7 @@ repository_url =
 repository_branch =
 launch_buttons = {}
 home_page_in_toc = False
+logo_only =
```

```suggestion
logo_only = False
```

this is giving the pre-commit check some problems so let's just explicitly set it to False as we do this with other bools in the config